decidim-core/lib/decidim/resourceable.rb

```diff
@@ -51,6 +51,7 @@ def sibling_scope(resource_name)
 
         scope = manifest.resource_scope(component)
         scope = scope.where("#{self.class.table_name}.id != ?", id) if manifest.model_class == self.class
+        scope = scope.not_hidden if manifest.model_class.respond_to?(:not_hidden)
```

Comment:
Can we add a spec for this 🙏🏽?  It should be in `decidim-proposals/spec/system/admin/view_proposal_details_from_admin_spec.rb` and/or `decidim-proposals/spec/system/proposals_spec.rb`, or any other place where you think it could be. 

We should also check that the related meeting doesn't appear in the admin select form. If not, I think that an admin could try to relate a (hidden) meeting with a proposal, and then they will not see it, and they'll think that's a bug.